src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -222,6 +249,37 @@ public override bool IsNumericColumn()
             return ret;
         }
 
+        public override BaseColumn FillNulls(object value, bool inPlace = false)
+        {
+            T TValue = (T)Convert.ChangeType(value, typeof(T));
```

Comment:
`TValue` - variables should be camel cased, so readers don't get confused that `TValue` is a generic type.